vscode-dotnet-runtime-library/src/Acquisition/IDotnetCoreAcquisitionWorker.ts

```diff
@@ -0,0 +1,18 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { IWindowDisplayWorker } from '../EventStream/IWindowDisplayWorker';
+import { IDotnetAcquireResult } from '../IDotnetAcquireResult';
+import { IExistingPath } from '../IExtensionContext';
+
+export interface IDotnetCoreAcquisitionWorker {
+    uninstallAll(): void;
+
+    resolveExistingPath(existingPaths: IExistingPath[] | undefined, extensionId: string | undefined, windowDisplayWorker: IWindowDisplayWorker): IDotnetAcquireResult | undefined;
```

Comment:
Hmm, shouldn't existing path's be compared against the currently requested version? Aka, if an extension that depends on the runtime extension "updates" and now requires a .NET 6 SDK where before they only required a 3.1 SDK how does this "resolve existing path" cross-check that the existing path version is correct? Very possible I'm misunderstanding this API but wanted to double check

Reply:
For now I'm not including the existing paths configuration in this PR since it's relatively complex to ensure we're resolving a correct version and we don't particularly want the SDK extension to be used on machines that have global installs anyways. I can definitely reincorporate this, but I would prefer to do it as a separate PR (if nothing else, to make this one less massive :))

Reply:
Ahhhh, this API is specific to SDK resolution then? In particular it's meant to resolve other SDKs on the path? If that's the case I'm not sure it makes sense for that type of orthogonal API to exist on the worker. Also 100% agree that it makes more sense in another PR

Reply:
It's specific to the existing paths configuration value, which right now is only supported for runtimes. It basically just checks to see if the path(s) the user provided in their configuration are valid and if so returns the one we want. Yeah, that's a fair point, it might be better outside of the acquisition worker.